hardware_interface/package.xml

```diff
@@ -16,7 +17,7 @@
   <author>Adolfo Rodriguez Tsouroukdissian</author>
 
   <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>boost</depend>
```

Comment:
perhaps only build dep here?

Reply:
It's used in `hardware_inteface/internal/interface_manager.h`, which is included by `hardware_interface/robot_hw.h`

Reply:
Just to clarify my suggestion :) 
```suggestion
  <build_depend>boost</build_depend>
```